apps/rehabstod/src/store/api.ts

```diff
@@ -142,6 +142,14 @@ export const api = createApi({
       }),
       providesTags: ['SickLeavePatient'],
     }),
+    getLUCertificatesForPatient: builder.query<LUCertificatesInfo, { encryptedPatientId: string }>({
+      query: (request) => ({
+        url: 'certificate/lu/person',
+        method: 'POST',
+        body: request,
+      }),
+      providesTags: ['LUCertificatesPatient'],
```

Comment:
We never cache the value from getSickLeavePatient by adding the ```keepUnusedDataFor```-flag. I think we should do the same for this request, so that we always get "latest and greatest"